.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish
+
+on:
+  workflow_run:
+    workflows: [Python]
+    types: [completed]
+    branches:
+      - "v[0-9]+.[0-9]+.x"
+  
+jobs:
+  copy-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python setup.py install && rm -rf build dist
+      - name: Build Wheels
+        run: |
+          python setup.py sdist bdist_wheel
+      - name: Check build
+        run: |
+          twine check dist/*
+      - name: Publish package
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
```

Comment:
@lionsardesai Just to make sure, this workflow is intended to publish in test PyPI, right? Is the plan to still do manual publication on official PyPI?

Reply:
If so, do we want to make it clear in the workflow name? Like `publish-test-pypi` or something

Reply:
yes for starters i plan on only publishing to test pypi.. if we do not find any issues with that for a while we'll start doing publishes to official pypi as well. Till then, yes it will be a manual step. I assume after maybe two releases we'll be more comfortable automating it?

Reply:
I see, yeah, that makes sense!